tests/main/res/TestIIDM_hvdc_diagrams_flat_start.iidm

```diff
@@ -0,0 +1,567 @@
+      <load id="ULYSSP6_ULYSS4" name="ULYSSP6_ULYSS4" loadType="UNDEFINED" p0="0" q0="0" connectableBus="ULYSSP6_S_VL6_TN1" p="0" q="0"/>
+      <shunt id="ULYSSP6_COND.11" name="ULYSSP6_COND.11" bPerSection="0.0015802469570189714" maximumSectionCount="1" currentSectionCount="1" bus="ULYSSP6_S_VL6_TN1" connectableBus="ULYSSP6_S_VL6_TN1" q="-84.933658677342095"/>
+    </voltageLevel>
+  </substation>
+  <substation id="AGAMEP6_S" name="AGAMEP6" country="FR" tso="RTE">
+    <voltageLevel id="AGAMEP6_S_VL6" name="AGAMEP6" nominalV="225" lowVoltageLimit="220" highVoltageLimit="245" topologyKind="BUS_BREAKER">
+      <busBreakerTopology>
+        <bus id="AGAMEP6_S_VL6_TN4" v="230.1670980570801" angle="-14.90222532511196"/>
+        <bus id="AGAMEP6_S_VL6_TN3" v="230.1670980570801" angle="-14.90222532511196"/>
+        <bus id="AGAMEP6_S_VL6_TN2" v="230.1670980570801" angle="-14.90222532511196"/>
+        <bus id="AGAMEP6_S_VL6_TN1" v="230.1670980570801" angle="-14.90222532511196"/>
+        <switch id="AGAMEP6_SW_AGAME___6TRO_2AB__DJ" kind="BREAKER" retained="true" open="false" bus1="AGAMEP6_S_VL6_TN4" bus2="AGAMEP6_S_VL6_TN2"/>
+        <switch id="AGAMEP6_SW_AGAME___6TRO_1AB__DJ" kind="BREAKER" retained="true" open="false" bus1="AGAMEP6_S_VL6_TN2" bus2="AGAMEP6_S_VL6_TN1"/>
+        <switch id="AGAMEP6_SW_AGAME___6COUPL_B__DJ" kind="BREAKER" retained="true" open="false" bus1="AGAMEP6_S_VL6_TN4" bus2="AGAMEP6_S_VL6_TN2"/>
+        <switch id="AGAMEP6_SW_AGAME___6COUPL_A__DJ" kind="BREAKER" retained="true" open="false" bus1="AGAMEP6_S_VL6_TN3" bus2="AGAMEP6_S_VL6_TN1"/>
+      </busBreakerTopology>
+      <load id="AGAMEP6_AGAME1" name="AGAMEP6_AGAME1" loadType="UNDEFINED" p0="200.91743567029621" q0="20.183907978163784" bus="AGAMEP6_S_VL6_TN1" connectableBus="AGAMEP6_S_VL6_TN1" p="201.87535430428619" q="20.376829337740514"/>
+      <load id="AGAMEP6_AGAME4" name="AGAMEP6_AGAME4" loadType="UNDEFINED" p0="0" q0="0" connectableBus="AGAMEP6_S_VL6_TN1" p="0" q="0"/>
+    </voltageLevel>
+  </substation>
+  <substation id="PRIAMP6_S" name="PRIAMP6" country="FR" tso="RTE">
+    <voltageLevel id="PRIAMP6_S_VL6" name="PRIAMP7" nominalV="225" lowVoltageLimit="220" highVoltageLimit="245" topologyKind="BUS_BREAKER">
+      <busBreakerTopology>
+        <bus id="PRIAMP6_S_VL6_TN2" v="231.80452291893261" angle="-19.394143257823384"/>
+        <bus id="PRIAMP6_S_VL6_TN1" v="231.80452291893261" angle="-19.394143257823384"/>
+        <switch id="PRIAMP6_SW_Disj_Couplage_1_2" kind="BREAKER" retained="true" open="false" bus1="PRIAMP6_S_VL6_TN2" bus2="PRIAMP6_S_VL6_TN1"/>
+      </busBreakerTopology>
+      <load id="PRIAMP6_PRIAM1" name="PRIAMP6_PRIAM1" loadType="UNDEFINED" p0="221.65287835010776" q0="20.30165226482961" bus="PRIAMP6_S_VL6_TN1" connectableBus="PRIAMP6_S_VL6_TN1" p="222.85228413011572" q="20.521958901656941"/>
+      <load id="PRIAMP6_PRIAM4" name="PRIAMP6_PRIAM4" loadType="UNDEFINED" p0="0" q0="0" connectableBus="PRIAMP6_S_VL6_TN1" p="0" q="0"/>
```

Comment:
I don't think we should have p, q values in the input iidm file

Reply:
I delete all p and q in the file then.